seqseek/chromosome.py

```diff
@@ -27,7 +28,12 @@ def __init__(self, chromosome_name, assembly=BUILD37, loop=False):
         """
         self.name = str(chromosome_name)
         self.assembly = assembly
-        self.loop = loop
+        if loop is None:
+            self.loop = True if self.name in MITOCHONDRIA_NAMES else False
```

Comment:
Unfortunately this is a potentially breaking change for clients of SeqSeek. When we added the `loop` convenience kwarg we defaulted it to False to prevent unintended consequences to clients. 

You're addition of `MITOCHONDRIA_NAMES` in this PR makes this easier to deal with though: clients can pass `loop = chromosome in MITOCHONDRIA_NAMES`. Is that sufficient?

Reply:
We'll keep MITOCHONDRIA_NAMES in lib.py, and recognize any of them in validate_loop, but otherwise loop defaults to False as before and requires the caller to set it explicitly.